roles/ckan/templates/ckan/ckan_production.ini

```diff
@@ -195,6 +195,7 @@ ckan.datapusher.url = {{ ckan_datapusher_url }}
 {% if ckan_datapusher_api_token %}
 ckan.datapusher.api_token = {{ ckan_datapusher_api_token }}
 {% endif %}
+ckan.datapusher.callback_url_base = http://ckan:5000/
```

Comment:
Unfortunately I am still getting this error when uploading files to the datastore: 
![image](https://github.com/fjelltopp/fjelltopp-ansible/assets/8988344/1c279ae6-3f64-4029-b647-b5e4a25ed22a)

Is that related to this setting?

Reply:
I need to add also documentation on confluence about setting up permissions to datastore. You need to login to ckan pod and run command `ckan datastore set-permissions` and copy output of that. After that open db pod, login to database and execute output of the previous command. More about it here: https://docs.ckan.org/en/2.9/maintaining/datastore.html#set-up-the-database under section "Set permissions".

PS. We can't do this over ssh cause we don't have a user with proper UID 🥲